### ISSUE TITLE
MTL / material design lite

### ISSUE THREAD

**Issue by Schwertspize** [papyros/papyros.github.io#39]
Google announced [Material Design Lite] (http://www.getmdl.io/). do we want to use it?

**Comment by hakarune** [comment]
Where would we add it? I don't see anything on the website that would be different with that setup...

**Comment by Schwertspize** [comment]
yes, the output would be mostly the same, but it's a framework and maybe we would think about using it......

**Comment by VirusKA** [comment]
Materialize is working for us so far. We'll prob keep it till Papyros is production reasdy
